frontend/test/resolution/testCallInitDeinit.cpp

```diff
@@ -1707,6 +1707,86 @@ static void test22() {
       });
 }
 
+static void test23a() {
+  // Ensure the copy-elided in formal doesn't error for its final use.
+  testActions("test23a",
+      R"""(
+      module M {
+        record Foo {}
+
+        proc doSomething(in x) {
+          return doSomethingElse(x);
+        }
+
+        proc doSomethingElse(in x) {
+          return 1;
+        }
+
+        proc test() {
+          var f : Foo;
+          var x = doSomething(f);
+        }
+      }
+      )""", {
+        {AssociatedAction::DEFAULT_INIT, "f",          ""}
```

Comment:
Is there a ``DEINIT`` to test here, or is that a separate bug? Same for the cases below.

Reply:
I believe there's no deinit in `test` to pick up because the passed argument is copy elided. Similar in existing `test16c` where we don't have a deinit for copy-elided `in` arg.